hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetImpl.java

```diff
@@ -2091,14 +2092,16 @@ public void unfinalizeBlock(ExtendedBlock b) throws IOException {
    * @param info the replica that needs to be deleted
    * @return true if data for the replica are deleted; false otherwise
    */
-  private boolean delBlockFromDisk(ReplicaInfo info) {
+  private boolean delBlockFromDisk(ReplicaInfo info, String bpid) {
     
     if (!info.deleteBlockData()) {
-      LOG.warn("Not able to delete the block data for replica " + info);
+      LOG.warn("Not able to delete the block data for replica " + info +
```

Comment:
can update warn("{}", arg) format?

Reply:
@haiyang1987 , Sir, thanks a lot for reviewing.  All opinions have fixed. please review again ~